Content: unfettered intro (#10248)
endless-sky/endless-sky: data/hai/midas.txt

# Copyright (c) 2024 by Hurleveur
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE. See the GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License along with
# this program. If not, see <https://www.gnu.org/licenses/>.

> outfit "Gold Bars"

Author: special outfits dont seem to have one?

	category "Special"
	plural "Gold Bars"
	cost 64000
	thumbnail "outfit/gold bars"
	mass 2.1
	"installable" -1

> 	description "At first it seems that these gold bars shine, as one would expect, but you can't help but feel like they also absorb light. The text 'nemo me impune lacessit' is written on the gold bars. Most outfitters do not truly understand the value these hold, however, and would not give you any more than the price of brute gold for it."

Reviewer: ```suggestion
	description `At first it seems that these gold bars shine, as one would expect, but you can't help but feel like they also absorb light. The text "Nemo Me Impune Lacessit" is written on the gold bars. Most outfitters do not truly understand the value these hold, however, and would not give you any more than the price of brute gold for it.`
```

> 	description "It shines, although you can't help but feel like it also absorbs light. For some reason, the text 'nemo me impune lacessit' is written on the gold bars. You wonder what that could mean... surely nothing to worry about. Most outfitters are not real 'connoisseurs' of the value these hold however, and would not give you any more than the price of brute gold for it."

Reviewer 2: ```suggestion
	description "These gold bars shine like any other, although you can't help but feel like they also absorb light. The text 'nemo me impune lacessit' is written on the gold bars. Most outfitters do not truly understand the value these hold, however, and would not give you any more than the price of brute gold for it."
```

Author: That feels of to me
Mb its just missing an 's' at others
Or mb smth like "These gold bars shine as one would expect" or smth to that effect

Reviewer 2: Is this okay?
```suggestion
	description "These gold bars shine as one would expect, although you can't help but feel like they also absorb light. The text 'nemo me impune lacessit' is written on the gold bars. Most outfitters do not truly understand the value these hold, however, and would not give you any more than the price of brute gold for it."
```

Author: ```suggestion
	description "At first it seems that these gold bars shine, as one would expect, but you can't help but feel like they also absorb light. The text 'nemo me impune lacessit' is written on the gold bars. Most outfitters do not truly understand the value these hold, however, and would not give you any more than the price of brute gold for it."
```
like this? caus do they really shine, even?
